mix.exs

```diff
@@ -79,14 +79,14 @@ defmodule Antikythera.Mixfile do
       {:fast_xml    , "1.1.32", [antikythera_internal: true]},
       {:recon       , "2.3.6" , [antikythera_internal: true]},
       {:relx        , "3.23.1", [antikythera_internal: true, only: :prod, runtime: false]}, # only to generate release
+      {:foretoken   , "0.2.0" , [antikythera_internal: true]},
```

Comment:
Thanks for this change.
It'd be nicer if `:foretoken` is placed somewhere above `:recon` and `:relx`, as, you know, these 2 hex packages are a bit special.

Reply:
Fixed in a4f176fc8b066a78b3351ad1132c3d7759203d9e